<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkForm</class>
 <widget class="QWidget" name="NetworkForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="hostname">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ip">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>212</y>
     <width>145</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="mask">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>212</y>
     <width>145</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dns">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>244</y>
     <width>145</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Network Configuration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hostname</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>217</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>217</y>
     <width>40</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mask</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>250</y>
     <width>40</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DNS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="abortBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>540</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Gateway</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gateway">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>244</y>
     <width>145</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Interface for Management network&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>540</width>
     <height>105</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/icons/resources/if2.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="prevBtn">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vlanCheckBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>216</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VLAN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vlan">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>hostname</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>ip</tabstop>
  <tabstop>mask</tabstop>
  <tabstop>gateway</tabstop>
  <tabstop>dns</tabstop>
  <tabstop>vlanCheckBox</tabstop>
  <tabstop>vlan</tabstop>
  <tabstop>nextBtn</tabstop>
  <tabstop>prevBtn</tabstop>
  <tabstop>abortBtn</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
